CHANGELOG.rst

```diff
@@ -2,6 +2,44 @@
 Change history
 ==============
 
+2.6.0
+=====
+*February, 28, 2025*
+
+**New features**
+
+* [#233] Fix set of values for PartijIdentificatoren (ENUM)
```

Comment:
Same comment as in objects-api: https://github.com/maykinmedia/objects-api/pull/537#discussion_r1973648245